### ISSUE TITLE
Is There FlashLoan Non-Demo and OpenZeppelin issue

### ISSUE THREAD

**Issue by jsim77** [Multiplier-Finance/MCL-FlashloanDemo#10]
Dear Support,

I'm interested and have tried to run using Flashloan Demo, was wondering do you have a "Real or Non-Demo" Flashloan? Also, I did try to run MCL Smart Contracts and found out that the --> Import "OpenZeppelin-solidity/contracts" in Remix IDE is "Not Found".

Yes, I understand that it has changed to "OpenZeppelin-contracts/" - I did locate and direct to the source such as Safemath.sol and replace it in the Remix IDE (replaced the OpenZeppelin-solidity/contracts) but the error is still the same --> Pop Up the same old error of "OpenZeppelin-solidity/contracts".

I've tried many ways from Stack Exchange to replace it but still the same, I'm curious as it seems like the code forbids me to change to the latest Import OpenZeppelin.

Sorry as I've run out of ideas to solve this, perhaps you know the solution?

Looking forward to your response.

Thanks.

**Comment by EGDOUG** [comment]
@jsim77 Were you able to get the FL to work? I am interested in the process also,

**Comment by jsim77** [comment]
> @jsim77 Were you able to get the FL to work? I am interested in the process also,

No, it doesn't work. I give up plus I'm not an expert coder. Thanks.

**Comment by jsim77** [comment]
@EGDOUG No, it doesn't work. I give up plus I'm not an expert coder. Thanks

**Comment by EGDOUG** [comment]
> @EGDOUG No, it doesn't work. I give up plus I'm not an expert coder. Thanks

Thanks!

**Comment by SirFlickka** [comment]
Yes but we need better understanding of these issues that's in this video


https://youtu.be/_5k5Ky2mg-g
